quickstart.rst

```diff
@@ -182,7 +182,7 @@ A flux session can be also be started under `Slurm <https://github.com/chaos/slu
   srun: job 1136410 has been allocated resources
   $
 
-After broker wireup is completed, the Flux session starts an “initial program” on rank 0 broker. By default, the initial program is an interactive shell, but an alternate program can be supplied on the ``flux start`` command line. Once the initial program terminates, the Flux session is considered complete and brokers exit.
+After broker wire up is completed, the Flux session starts an “initial program” on rank 0 broker. By default, the initial program is an interactive shell, but an alternate program can be supplied on the ``flux start`` command line. Once the initial program terminates, the Flux session is considered complete and brokers exit.
```

Comment:
flux-core docs uses `wireup` (it is added to our dictionary anyway). Up to you but maybe just add this one to the spell.en.pws here?